src/App.test.js

```diff
@@ -0,0 +1,151 @@
+    const numberOne = '1';
+    const numberTwo = '2';
+    const operation = 'x';
+
+    expect(operate(numberOne, numberTwo, operation)).toBe('2');
+  });
+
+  test('operate.js division test', () => {
+    const numberOne = '1';
+    const numberTwo = '2';
+    const operation = '÷';
+
+    expect(operate(numberOne, numberTwo, operation)).toBe('0.5');
+  });
+
+  test('operate.js division by 0 test', () => {
+    const numberOne = '1';
+    const numberTwo = '0';
+    const operation = '÷';
+
+    expect(operate(numberOne, numberTwo, operation)).toBe('Can\'t divide by 0.');
+  });
+
+  test('operate.js remainder test', () => {
+    const numberOne = '5';
+    const numberTwo = '2';
+    const operation = '%';
+
+    expect(operate(numberOne, numberTwo, operation)).toBe('1');
+  });
+
+  test('operate.js remainder when divide by 0 test', () => {
+    const numberOne = '1';
+    const numberTwo = '0';
+    const operation = '%';
+
+    expect(operate(numberOne, numberTwo, operation)).toBe('Can\'t find modulo as can\'t divide by 0.');
+  });
+});
+
+describe('component test', () => {
+  it('Calculator component should perform the calculations correctly on initial run', () => {
+    const tree = renderer.create(<Calculator />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Quote component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Quotes />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('CalculatorPage component should perform correctly on initial run', () => {
+    const tree = renderer.create(<CalculatorPage />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Home component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Home />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Buttons component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Buttons />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Last component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Last />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('NavBar component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Router><NavBar /></Router>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Header component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Router><Header /></Router>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Button component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Button />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  it('Screen component should perform correctly on initial run', () => {
+    const tree = renderer.create(<Screen />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test('Simulation of user clicking button 9 twice', () => {
```

Comment:
- [ ] Your test is failing, kindly fix this
![Screenshot from 2022-08-11 14-40-23](https://user-images.githubusercontent.com/66097371/184147683-5ae5ddfa-8d7c-4ba9-9058-c7b41c59bf39.png)